port/unix_include/omrsock.h

```diff
@@ -0,0 +1,37 @@
+/*******************************************************************************
+ * Copyright (c) 2019, 2019 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+
+#if !defined(OMRSOCK_H_)
+#define OMRSOCK_H_
+
+
+#include <stdio.h>
+#include <stdlib.h>
+
+#include "omrportsock.h"
+
+/* Per-thread buffer for platform-dependent socket information */
+typedef struct OMRSocketPTB {
```

Comment:
Yes, right now there aren't differences. I just put them in different files because it seems like in j9sock.c there were different definitions of the socket per thread buffer. I removed the definitions that I thought were unnecessary but kept them still in different files.

But yes, I will remove them for now. Should I also remove the `unix_include/omrsock.h` and `win32_include/omrsock.h` files as well? This is because in `omrsock.h`, the ptb's were the only definitions in them.

Reply:
Yes, we should remove them for now. These changes should appear incrementally as you implement the API. Currently, it's dead code since it is not used.

Reply:
Sounds good, I will do that.